docs/guides/getting-started-orchestrate-human-tasks.md

```diff
@@ -41,7 +41,7 @@ To create an automated process with user tasks, take the following steps:
 9. Append a gateway to your user task by dragging it onto the dashboard from the palette on the left side of the screen, or by clicking on the user task `Decide what's for dinner` and clicking on the element you'd like to create next. In this case, we've selected the diamond icon to create a gateway.
 10. Create two sequence flows (represented by the arrows) from the gateway and two new user tasks based on what the user decides to eat. In this case, we've named ours `Prepare chicken` and `Prepare salad`.
```

Comment:
I think exactly in step 10 we need a hint that expressions will be needed, otherwise this process cannot be deployed successfully.
And to create the expressions, you will need a variable name, that comes from the form eaelier on. This is IMHO not yet clear

Reply:
@akeller Thoughts here? Do you think pulling in expressions, callouts, etc, will make this too complex for a new user? Do we think pulling out the form will make things too thin? If the former sounds reasonable, I can plug in a few sentences referencing expressions.

Reply:
@berndruecker I've added some details on the creation of the form and affiliated expressions within my latest commit. Thoughts?